wicket-kendo-ui/src/main/java/com/googlecode/wicket/kendo/ui/widget/menu/MenuBehavior.java

```diff
@@ -53,7 +53,17 @@ public MenuBehavior(String selector)
 		super(selector, METHOD);
 	}
 
-	/**
+    /**
+     * Constructor
+     *
+     * @param selector the html selector (ie: "#myId")
+     */
+    protected MenuBehavior(String selector, String method)
```

Comment:
As it is for internal purpose only, wouldn't be better to have a package visibility only?